src/core/FileUpload/FileUploadTemplate.tsx

```diff
@@ -57,6 +63,7 @@ export const FileUploadTemplate = (props: FileUploadTemplateProps) => {
         id='file-input'
         onChange={onChange}
         multiple={acceptMultiple}
+        {...inputProps}
```

Comment:
@veekeys @bentleyvk Need thoughts on this. Mostly done to allow any arbitrary input attributes (e.g. [`capture`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/capture)) but it does make the `acceptMultiple` prop redundant.

The alternative is:
```suggestion
        accept={acceptType}
```

Reply:
Ugh, very hard decision... There is only 4 props related to file upload:
![image](https://user-images.githubusercontent.com/36186912/121638223-487bac80-ca93-11eb-9475-d5e391091757.png)
Maybe we can just manually have those because I don't know if other props are needed in this case.

Reply:
If its really only 4 props, maybe no need to go complicated way...
And at the end, i guess its not a bad deal to end up with a lot of props as long as they make sense and guarantee quality (implementation from our side).